diff --git a/contrib/libvpx/module.defs b/contrib/libvpx/module.defs
index 8f05a9d21..093b51ff2 100644
--- a/contrib/libvpx/module.defs
+++ b/contrib/libvpx/module.defs
@@ -20,6 +20,9 @@ LIBVPX.CONFIGURE.extra =     \
     --disable-docs              \
     --disable-unit-tests
 
+ifeq (on,$(GCC.lto))
+    LIBVPX.CONFIGURE.extra += --disable-strip
+endif
 
 ifeq (1,$(HOST.cross))
     LIBVPX.CONFIGURE.args.build =
